app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connections = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connections.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            "VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connections.commit()
+
+    def retrieve(self) -> list:
+        actors_cursor = self._connections.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
+        )
+
+        return [Actor(*row) for row in actors_cursor]
+
+    def update(self,
+               id: int,
+               first_name: str,
+               last_name: str) -> None:
+        self._connections.execute(
+            f"UPDATE {self.table_name} "
+            "SET first_name = ?, last_name = ?"
+            "WHERE id = ?",
+            (id, first_name, last_name)
```

Comment:
Does this query work as intended? It seems to me that `id` is passed to `first_name` field.
By the way, `id` is a reserved name in Python.

Reply:
Yes, works properly